src/components/common/Toolbar/Toolbar.tsx

```diff
@@ -32,21 +32,34 @@ import {
   previousItem,
 } from './utils';
 
-const ToolbarStyled = styled(MuiToolbar)(({ theme }) => ({
+const ToolbarStyled = styled(MuiToolbar)<{
+  toolbarDirection: ToolbarDirection;
+}>(({ theme, toolbarDirection }) => ({
   border: `1px solid ${theme.palette.divider}`,
   padding: 2,
   gap: 2,
   minHeight: 'auto',
   backgroundColor: theme.palette.background.default,
   borderRadius: theme.shape.borderRadius,
   boxShadow: theme.shadows[2],
+  flexDirection: toolbarDirection,
 }));
 
+export type ToolbarDirection = 'row' | 'column';
+
 export type ToolbarProps = PropsWithChildren<
-  Partial<React.ComponentPropsWithRef<'div'> & { sx?: SxProps<Theme> }>
+  Partial<
+    React.ComponentPropsWithRef<'div'> & { sx?: SxProps<Theme> } & {
+      toolbarDirection?: ToolbarDirection;
+    }
+  >
```

Comment:
The `Partial<>` was wrong before. This can be simplified to:

```suggestion
  Partial<React.ComponentPropsWithRef<'div'>> & {
    sx?: SxProps<Theme>;
    toolbarDirection?: ToolbarDirection;
  }
```